sui_programmability/examples/games_with_chance/sources/satoshi_flip.move

```diff
@@ -0,0 +1,264 @@
+module games_with_chance::satoshi_flip {
```

Comment:
Add license header (at the top of the file) and public docs for every function, i.e., see https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/sources/safe.move

For public docs use ///